databricks/koalas/generic.py

```diff
@@ -1435,8 +1568,115 @@ def var(self, axis=None, numeric_only=True) -> Union[Scalar, "Series"]:
         >>> df['a'].var()
         1.0
         """
+
+        def var(spark_column, spark_type):
+            if isinstance(spark_type, BooleanType):
+                spark_column = spark_column.cast(LongType())
+            elif not isinstance(spark_type, NumericType):
+                raise TypeError("Could not convert {} to numeric".format(spark_type.simpleString()))
+            return F.variance(spark_column)
+
+        return self._reduce_for_stat_function(var, name="var", axis=axis, numeric_only=numeric_only)
+
+    def median(
```

Comment:
This is just moved here for ease of type annotation.

Reply:
Where is `bool` overwritten?

Reply:
https://github.com/databricks/koalas/blob/5b1205b92c545733b0405753163d6ae90772977c/databricks/koalas/generic.py#L1885

Reply:
Got it!